apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java

```diff
@@ -143,4 +151,12 @@ private Optional<String> parseOperation(String cmd) {
         }
         return Optional.empty();
     }
+
+    private String showBatchCommands(Object argument) {
```

Comment:
use `CommandsData` instead of `Object`? Since you've already known it is a CommandsData

Reply:
Yes, indeed.